clients/src/main/java/org/apache/kafka/clients/consumer/KafkaShareConsumer.java

```diff
@@ -115,23 +115,26 @@
  * as naturally happens when the locks time out. This limit is controlled by the broker configuration property
  * {@code group.share.record.lock.partition.limit}. By limiting the duration of the acquisition lock and automatically
  * releasing the locks, the broker ensures delivery progresses even in the presence of consumer failures.
+ *
```

Comment:
nit: The style of the rest of this javadoc comment is not to have blank lines before the `<p>` and to have the `<p>` on a line by itself.